net/http/src/THttpServer.cxx

```diff
@@ -924,18 +828,19 @@ void THttpServer::ProcessRequest(THttpCallArg *arg)
          // if accepted, reply with connection id, which must be used in the following communications
          arg->SetMethod("WS_CONNECT");
 
-         if (handler && handler->HandleWS(arg)) {
-            arg->SetMethod("WS_READY");
-
-            TLongPollEngine *handle = new TLongPollEngine("longpoll", arg->fPathName.Data());
+         THttpLongPollEngine *handle = new THttpLongPollEngine("longpoll", arg->fPathName.Data());
```

Comment:
Please use a `unique_ptr`. I cannot tell who owns `handle` if `delete handle` isn't called, and the use of `unique_ptr` makes this visible (you have to `std::move` it to its new owner somewhere, and its new owner now *knows* that it owns it).

In general, new code should really never contain `new` (use `make_unique`).

Reply:
when used without new operator, make_unique is c++14. And THttpServer classes are still normal ROOT6.

Reply:
We have `<ROOT/RMakeUnique.hxx>` for that, also for C++11.